subgraphs/stargate/protocols/stargate/config/deployments/stargate-base/configurations.json

```diff
@@ -0,0 +1,18 @@
+{
+  "deployment": "STARGATE_BASE",
+  "network": "base",
```

Comment:
I think we are getting a build error bc this isn't the proper network name for the graph node

Reply:
You should be able to find it in the graph node release notes or source code

Reply:
problem was with the lpStaking dataSource naming, which is used in imports; fixed it.